src/components/builder/values/values-config.tsx

```diff
@@ -0,0 +1,37 @@
+import {useFormikContext} from 'formik';
+
+import ItemsExpression from './items-expression';
+import {SchemaWithDataSrc} from './types';
+import ValuesSrc from './values-src';
+import ValuesTable, {ValuesTableProps} from './values-table';
+
+export interface ValuesConfigProps<T> {
+  name: ValuesTableProps<T>['name'];
+}
+
+/**
+ * The `ValuesConfig` component allows a form builder to specify available options.
+ *
+ * Certain component types like dropdowns, radio fields and multi-option fields present
+ * a pre-configured list of available options to the end-user. This component is used to
+ * do this pre-configuration.
+ *
+ * Options can either be provided manually upfront, or they can be set dynamically by
+ * referencing other variables in the form evaluation context.
+ */
+export function ValuesConfig<T extends SchemaWithDataSrc>({name}: ValuesConfigProps<T>) {
+  const {
+    values: {
+      openForms: {dataSrc},
+    },
+  } = useFormikContext<T>();
+  return (
+    <>
+      <ValuesSrc />
```

Comment:
best would be to pass an `onChange` here so that we can reset the appropriate field

Reply:
I tried that and it leads to race conditions, so I'm now using `useLayoutEffect` instead.

There might be an option to use React 18's `flushSync` instead, but the builder & backend are still stuck on React 17 currently :(